package.json

```diff
@@ -54,7 +54,7 @@
     "remap-istanbul": "^0.9.5",
     "simple-mock": "^0.8.0",
     "tslint": "^5.2.0",
-    "typescript": "^2.3.4"
+    "typescript": "2.3.4"
```

Comment:
please move revert this. I know there is a problem with the iterable. I'll fix it on this PR. Thanks!